doc/man7/EVP_SIGNATURE-ML-DSA.pod

```diff
@@ -117,6 +127,39 @@ To sign a message using an ML-DSA EVP_PKEY structure:
         EVP_PKEY_CTX_free(sctx);
     }
 
+To calculate external mu using an ML-DSA EVP_PKEY structure:
+
+    void do_external_mu(EVP_PKEY *key, unsigned char *msg, size_t msg_len)
+    {
+        int mu_only = 1;
+        unsigned char *mu = NULL;
+        size_t mu_len = 0;
+        const OSSL_PARAM params[] = {
+            OSSL_PARAM_octet_string("context-string", (unsigned char *)"A context string", 16),
+            OSSL_PARAM_int("mu-only", &mu_only),
+            OSSL_PARAM_END
+        };
+        EVP_PKEY_CTX *sctx = EVP_PKEY_CTX_new_from_pkey(NULL, pkey, NULL);
+        EVP_SIGNATURE *sig_alg = EVP_SIGNATURE_fetch(NULL, "ML-DSA-44", NULL);
+
+        EVP_PKEY_sign_message_init(sctx, sig_alg, params);
```

Comment:
Some commentary above here may be helpful:

- the context string is of course optional, and typically not required.
- The "mu-only" parameter (`OSSL_SIGNATURE_PARAM_MU_ONLY`) is essential to obtain a `μ` value rather than a full signature.